core/src/main/java/com/google/zxing/datamatrix/DataMatrixWriter.java

```diff
@@ -159,17 +160,33 @@ private static BitMatrix encodeLowLevel(DefaultPlacement placement, SymbolInfo s
    * @param matrix The input matrix.
    * @return The output matrix.
    */
-  private static BitMatrix convertByteMatrixToBitMatrix(ByteMatrix matrix) {
-    int matrixWidgth = matrix.getWidth();
+  private static BitMatrix convertByteMatrixToBitMatrix(ByteMatrix matrix, int reqWidth, int reqHeight) {
```

Comment:
It's worth a little javadoc to clarify what these two args are, especially as they're abbreviated